e2e/cases/plugin-api/plugin-hooks/index.test.ts

```diff
@@ -57,6 +63,9 @@ const createPlugin = () => {
       api.onDevCompileDone(() => {
         names.push('OnDevCompileDone');
       });
+      api.onDevCompileEnvironmentDone(() => {
```

Comment:
`onDevCompileEnvironmentDone` is too long, maybe we can find a shorter name

Reply:
how about `onBeforeEnvironmentCompile` and `onAfterEnvironmentCompile`?  It will be triggered during dev and build.

Reply:
After comparing, I now tend to prefer using `onBeforeEnvironmentCompile` and `onAfterEnvironmentCompile`. They are shorter and clearer.